test/e2e_flags.go

```diff
@@ -45,45 +45,62 @@ func initializeServingFlags() *ServingEnvironmentFlags {
 	var f ServingEnvironmentFlags
 
 	// Only define and set flags here. Flag values cannot be read at package init time.
-	flag.BoolVar(&f.ResolvableDomain,
-		"resolvabledomain",
-		false,
-		"Set this flag to true if you have configured the `domainSuffix` on your Route controller to a domain that will resolve to your test cluster.")
-
-	flag.BoolVar(&f.Https,
-		"https",
-		false,
-		"Set this flag to true to run all tests with https.")
-
-	flag.StringVar(&f.IngressClass,
-		"ingressClass",
-		network.IstioIngressClassName,
-		"Set this flag to the ingress class to test against.")
-
-	flag.StringVar(&f.CertificateClass,
-		"certificateClass",
-		network.CertManagerCertificateClassName,
-		"Set this flag to the certificate class to test against.")
-
-	flag.IntVar(&f.Buckets,
-		"buckets",
-		1,
-		"Set this flag to the number of reconciler buckets configured.")
-
-	flag.IntVar(&f.Replicas,
-		"replicas",
-		1,
-		"Set this flag to the number of controlplane replicas being run.")
-
-	flag.BoolVar(&f.EnableAlphaFeatures,
-		"enable-alpha",
-		false,
-		"Set this flag to run tests against alpha features")
-
-	flag.BoolVar(&f.EnableBetaFeatures,
-		"enable-beta",
-		false,
-		"Set this flag to run tests against beta features")
+	if flag.Lookup("resolvabledomain") == nil {
```

Comment:
this seems indicative of some larger problem 🤔 

If we skip flags that were initialized with another variable, then our variable won't get set, so I don't think this is right...

Reply:
Yeah, that's what I am trying to fix.
But a larger problem is that somehow the kube flags aren't set...